testsuite/features/step_definitions/command_steps.rb

```diff
@@ -1420,6 +1420,16 @@
   $server.run(command)
 end
 
+When(/^I refresh the packages list via package manager on "([^"]*)"$/) do |host|
+  node = get_target(host)
+  if host.include? 'ceos'
+    cmd_clean = "yum -y clean all"
+    cmd_cache = "yum -y makecache"
+  end
+  node.run(cmd_clean)
+  node.run(cmd_cache)
```

Comment:
```suggestion
  next unless host.include? 'ceos'
    
  node.run('yum -y clean all')
  node.run('yum -y makecache')
```

Reply:
Also, is this step needed at all ? Isn't `yum` doing a refresh automatically at each operation?

Reply:
If so, if we want this step to be only used for RH-compatible hosts, we should somehow clarify the step definition text.

For instance:
`When(/^I refresh the packages list via YUM package manager on "([^"]*)"$/) do |host|`

Reply:
I wanted to leave it unspecific for future features. Maybe we want to include Debian/Ubuntu hosts for some reasons as well.